recipe/meta.yaml

```diff
@@ -33,6 +44,7 @@ build:
   skip: true  # [cuda_compiler_version == "10.2"]
   skip: true  # [cuda_compiler_version == "11.0"]
   skip: true  # [cuda_compiler_version == "11.1"]
+  skip: true  # [cuda_compiler_version != "12.0"]
```

Comment:
Guessing this is a temporary measure to reign in builds offline. Is that right?

Reply:
yes. I can add a note about it. but I'm not thrilled about using 8OSX CPUs for 8-12 hours for this.

Reply:
Do you mean the CI CPUs? If so, can't we stop the Azure builds manually? Or do you need help with local builds?

Reply:
I don't know how to stop the build manually. I just feel that somebody will remember to enable the osx builds.....

Reply:
Agreed :)